doc/api/tty.md

```diff
@@ -88,6 +88,8 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* Unrealiable event handler execution on all Windows platforms.
```

Comment:
I could add something like
`Event handler only is only executed when the width is changed`

It was also [suggested here](https://github.com/nodejs/node/issues/13197#issuecomment-307073097) that raw mode must be enabled but I did not find this in my own testing.